### PR TITLE
TL Localization - Chapter 6, section 2 - part 3 (1,397 words)

### PR DESCRIPTION
**Pull request by heywens**

### CODE REVIEW

File: book/06-github/sections/2-contributing.asc [hunk 1]
```diff
@@ -260,160 +260,161 @@ Isa sa mga magagandang bagay tungkol sa Git ay maaari mong patuloy na gawin iyon
 
 Kung talagang gusto mong i-rebase ang branch upang malinis ito, maaari mong tiyak na gawin ito, ngunit ito ay lubos na hinihikayat na huwag piliting mag-push sa branch na ang Kahilingan na Pull ay binuksan na. Kung ang ibang tao ay naka-pull na dito at nakagawa ng mas maraming trabaho, mararanasan mo ang lahat ng isyo na nakabalangkas sa <<_git_branching#_rebase_peril>>. Sa halip, i-push ang branch na naka-rebase sa isang bagong branch sa GitHub at magbukas ng isang bagong Kahilingan na Pull na tumutukoy sa luma, pagkatapos ay isara ang orihinal.
 
-===== References
+===== Mga Reperensiya
 
-Your next question may be ``How do I reference the old Pull Request?''. It turns out there are many, many ways to reference other things almost anywhere you can write in GitHub.
+Ang iyong susunod na tanong ay maaaring ``Paano ko ireperensiya ang lumang Kahilingan na Pull?''. Lumilitaw na mayroong maraming, maraming mga paraan upang magamit ang iba pang mga bagay halos kahit saan maaari kang sumulat sa GitHub.
```
Comment (cherryloua) [inline book/06-github/sections/2-contributing.asc:266]: "Lumilitaw na mayroong maraming, maraming mga paraan" I suggest to make it "marami-raming mga paraan" I think its sounds good to hear. lol

File: book/06-github/sections/2-contributing.asc [hunk 2]
```diff
@@ -260,160 +260,161 @@ Isa sa mga magagandang bagay tungkol sa Git ay maaari mong patuloy na gawin iyon
...
 
-Let's look at an example. Say we rebased the branch in the previous example, created a new pull request for it, and now we want to reference the old pull request from the new one. We also want to reference an issue in the fork of the repository and an issue in a completely different project. We can fill out the description just like <<_pr_references>>.
+Tingnan natin ang isang halimbawa. Sabihing ating na-rebase ang branch sa nakaraan halimbawa, naglikha ng isang bagong kahilingan na pull para rito, at ngayon gusto natin na ireperensiya ang lumang kahilingan na pull mula sa bago. Gusto din natin na ireperensiya ang isang isyu sa fork ng epositoryo at isyu sa ganap na naiibang proyekto. Maaari nating punan ang paglalarawan kagaya ng <<_pr_references>>.
```
Comment (cherryloua) [inline book/06-github/sections/2-contributing.asc]: "na-rebase ang branch sa nakaraan halimbawa" to "nakraang halimbawa"

Comment (cherryloua) [reply book/06-github/sections/2-contributing.asc]: "fork ng epositoryo at isyu" to "fork ng repositoryo at isyu" a slight typo.

File: book/06-github/sections/2-contributing.asc [hunk 3]
```diff
@@ -260,160 +260,161 @@ Isa sa mga magagandang bagay tungkol sa Git ay maaari mong patuloy na gawin iyon
...
-.Cross references rendered in a Pull Request.
+.Mga pagtukoy na naibigay sa isang Kahilingan na Pull.
 image::images/mentions-02-render.png[PR references rendered]
 
-Notice that the full GitHub URL we put in there was shortened to just the information needed.
+Pansinin na ang buong URL ng GitHub na ating nilagay doon ay pinaikli sa kailangan lamang na impormasyon.
 
-Now if Tony goes back and closes out the original Pull Request, we can see that by mentioning it in the new one, GitHub has automatically created a trackback event in the Pull Request timeline. This means that anyone who visits this Pull Request and sees that it is closed can easily link back to the one that superseded it. The link will look something like <<_pr_closed>>.
+Ngayon kung babalik si Tony at isasara ang orihinal na Kahilingan na Pull, maaari nating makita na sa pagbabanggit nito sa isang bago, awtomatikong nilikha ng GitHub ang isang trackback event sa timeline ng Kahilingan na Pull. Ito ay nangangahilugan na sinuman ang bibisita sa Kahilingan na Pull na ito at makakakita na ito ay naisara ay maaaring madaling maka-link pabalik sa pumalit nito. Magiging tulad ng <<_pr_closed>> ang link.
```
Comment (cherryloua) [inline book/06-github/sections/2-contributing.asc]: "Ito ay nangangahilugan na sinuman" change to "Ito ay nangangahulugan na simuman" minimal typo.

File: book/06-github/sections/2-contributing.asc [hunk 4]
```diff
@@ -260,160 +260,161 @@ Isa sa mga magagandang bagay tungkol sa Git ay maaari mong patuloy na gawin iyon
...
-See <<_example_markdown>> for an example of how comments or text can be written and then rendered using Markdown.
+Tingnan ang <<_example_markdown>> para sa isang halimbawa kung paano masusulat ang mga komento at teksto at pagkatapos ay gawin gamit ang Markdown.
 
 [[_example_markdown]]
-.An example of GitHub Flavored Markdown as written and as rendered.
+.Isang halimbawa ng Pinalasang Markdown ng GitHub na nakasulat at nagawa.
 image::images/markdown-01-example.png[Example Markdown]
 
-The GitHub flavor of Markdown adds more things you can do beyond the basic Markdown syntax. These can all be really useful when creating useful Pull Request or Issue comments or descriptions.
+Ang timpla ng Markdown ng GitHub ay nagdadagdag ng mas maraming bagay na maaari mong gawin lampas sa pangunahing Markdown syntax. Ang mga ito ay maaari magagamit kapag naglilikha ng kapaki-pakinabang na Kahilingan na Pull o mga komento o paglalarawan ng Isyu.
```
Comment (cherryloua) [inline book/06-github/sections/2-contributing.asc]: "ito ay maaari magagamit" change to "ito ay maaaring magagamit"

**Review by cherryloua — COMMENTED** [review]
Hi Miss @heywens, 

I have a few suggestions to improve your contribution. Cheers.

File: book/06-github/sections/2-contributing.asc [hunk 5]
```diff
@@ -260,160 +260,161 @@ Isa sa mga magagandang bagay tungkol sa Git ay maaari mong patuloy na gawin iyon
...
 for(int i=0 ; i < 5 ; i++)
 {
-   System.out.println("i is : " + i);
+   System.out.println("i ay : " + i);
 }
 ```
 ----
 
-If you add a language name like we did there with 'java', GitHub will also try to syntax highlight the snippet. In the case of the above example, it would end up rendering like <<_md_code>>.
+Kung nagdagdag ka ng isang pangalan ng wika tulad ng ginawa natin doon sa 'java', susubukan din ng GitHub na i-higlight ang syntax ng snippet. Sa kaso sa itaas na halimbawa, ito ay matatapos sa paggawa tulad ng <<_md_code>>.
```
Comment (cherryloua) [inline book/06-github/sections/2-contributing.asc]: "GitHub na i-higlight ang syntax" change to "GitHub na i-highlight ang syntax" slight typo

File: book/06-github/sections/2-contributing.asc [hunk 6]
```diff
@@ -260,160 +260,161 @@ Isa sa mga magagandang bagay tungkol sa Git ay maaari mong patuloy na gawin iyon
...
+Kapag nagawa, ito ay magiging tulad ng <<_md_emoji>>.
 
 [[_md_emoji]]
-.Heavy emoji commenting.
+.Mabibigay na pagkokomento na emoji.
 image::images/markdown-07-emoji.png[Emoji]
 
-Not that this is incredibly useful, but it does add an element of fun and emotion to a medium that is otherwise hard to convey emotion in.
 
+Hindi na ito ay hindi kapani-paniwala kapaki-pakinabang, ngunit ito ay nagdaragdag ng isang elemento ng kasiyahan at damdamin sa isang daluyan na kung hindi man ay mahirap upang ihatid ang damdamin.
```
Comment (cherryloua) [inline book/06-github/sections/2-contributing.asc]: "Hindi na ito ay hindi kapani-paniwala" i think it sounds better if we do it like this "Hindi sa ito ay hindi kapani-paniwala"

File: book/06-github/sections/2-contributing.asc [hunk 7]
```diff
@@ -260,160 +260,161 @@ Isa sa mga magagandang bagay tungkol sa Git ay maaari mong patuloy na gawin iyon
...
+Mayroon talagang maraming bilang ng mga serbisyo ng web na gumagamit sa mga araw na ito ng mga karakter na emoji. Isang magaling na sheet ng panlilinlang na magamit upang mahanap ang emoji na nagpapahayag kung ano ang gusto mong sabihin ay matatagpuan sa:
 
 http://www.emoji-cheat-sheet.com
 ====
 
-===== Images
+===== Mga Larawan
 
-This isn't technically GitHub Flavored Markdown, but it is incredibly useful. In addition to adding Markdown image links to comments, which can be difficult to find and embed URLs for, GitHub allows you to drag and drop images into text areas to embed them.
+Hindi ito Pinalasang Markdown ng GitHub, ngunit ito ay hindi kapani-paniwala kapaki-pakinabang. Bilang karagdagan sa pagdaragdag ng mga link ng larawan ng Markdown sa mga komento, na maaaring magin mahirap na mahanap at para ma-embed ang mga URL, ang GitHub ay nagpapahintulot sa iyo na i-drag at ihulog ang mga larawan sa mga text area upang ma-embed ang mga ito.
```
Comment (cherryloua) [inline book/06-github/sections/2-contributing.asc]: "ito ay hindi kapani-paniwala kapaki-pakinabang" I thinki it it better to say it like this  "ito ay lubhang kapaki-pakinabang"

**Comment by heywens** [comment]
Hi @cherryloua, thank you for pointing out those typos. Suggestions on changing some phrases have been also applied. Thank you.

**Pull request merged**